Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md

```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
+
+To use this integration for testing:
+1. Create an instance - Add a listening port and mark the *Long running instance* checkbox.
+2. Take the server with the listening port that has been configured in the integration settings,
+and configure the **Workday_IAM** instance with this url and invoke the *fetch_incidents*.
```

Comment:
2. Take the server URL and listening port that you entered in the integration settings and configure the **Workday_IAM** instance and invoke *fetch_incidents*. 

@Shellyber maybe we should put an example of the URL and port? I know it's basic, but couldn't hurt.